packages/components/e2e/component-objects/src/main/java/org/talend/component/Component.java

```diff
@@ -67,17 +75,16 @@ public Component(WebDriver driver, String name, WebElement root) {
      * Get component from its selector
      *
      * @return WebElement found with selector
-     * @throws NotFoundException if no elements are found or if more than one element are found
      */
-    public WebElement getElement() throws NotFoundException {
+    public WebElement getElement() {
```

Comment:
Why did you remove that ?

Reply:
Because this exception is a RuntimeException so this code is useless